ovos-i2csound

```diff
@@ -38,8 +38,52 @@ detect_i2c_device() {
     fi
 }
 
+pulseaudio_config() {
+    echo "configuring pulseaudio"
+    # Set default configurations
+    if [ -f /var/lib/alsa/default-asound.state ]; then
```

Comment:
Playing around with the .state files is no longer needed as the "amixer -c " commands are setting up the different alsa controls now.

This will only make things very nasty for users with slioghtly different hardware as the .state file are full backup of all alsa control on that system.

Its the reason I implemented the amixer commands. That way you only setup the control for that specific card.

Reply:
This was there as a default and I could not get pulse to start without it.  That is the ONLY state file installed.  The rest are set with `amixer`.

Reply:
I also thought that in order for alsa to use pulse it needed the file in `etc/asound.conf` directing it to do so?

Reply:
All of this is working without overlaying state files for me in [neon-debos](https://github.com/NeonGeckoCom/neon_debos/pull/101)

Reply:
The alsa .state files are written/read by alsactl system. 

That can be run as daemon on boot or after changing alsa stuff.
https://man.archlinux.org/man/alsactl.1.en

PulseAudio should not have any links to it other then pulling in the stuff from the pulseaudio bridge / alsa-plugins.